### ISSUE TITLE
Add NameLayer messages

### ISSUE THREAD

**Issue by Maxopoly** [TealNerd/CivGuide#11]
There should be one if a person gets invited to a group and possibly also one, once the person sends his first chat message to link to more docu.

**Comment by TealNerd** [comment]
Yeah I want to do this but right now there's no InviteEvent in Namelayer.
